docs/user-docs/apps.md

```diff
@@ -2,45 +2,83 @@
 
 A list of apps with support for Jellyfin.
 
-⚠ **Note** - Many of the official apps were forked from Emby and are not currently building and/or functional.
+**Note**: Many of the official apps were forked from Emby and are not currently building and/or functional.
 If you interested in helping out, please see our [contribution guide](/developer-docs/contributing).
 
 ### Android
 
-- **[Jellyfin for Android](https://github.com/jellyfin/jellyfin-android)** - The official Jellyfin Android app.
-- **[Yatse](https://www.yatse.tv/)** - An Android remote control for Jellyfin with support for Chromecasts.
+**[Jellyfin for Android](https://github.com/jellyfin/jellyfin-android)**
+
+The official Jellyfin Android app.
+Status: Unreleased but In Development
+
+**[Yatse](https://www.yatse.tv/)**
+
+An Android remote control for Jellyfin with support for Chromecasts.
+Status: In Development
 
 ### Android TV
 
-- **[Jellyfin for Android TV](https://github.com/jellyfin/jellyfin-androidtv)** - The official Jellyfin Android TV app.
+**[Jellyfin for Android TV](https://github.com/jellyfin/jellyfin-androidtv)**
+
+The official Jellyfin Android TV app.
+Status: Unreleased but In Development
 
 ### Desktop
 
-- **[Jellyfin Theater (Electron)](https://github.com/jellyfin/jellyfin-theater-electron)** - A desktop media interface for Windows, macOS, and Linux.
-- **[Jellyfin Theater (Raspberry Pi)](https://github.com/jellyfin/jellyfin-theater-pi)** - A desktop media interface built for Raspberry Pi's.
-- **[Jellyfin Theater (Windows)](https://github.com/jellyfin/jellyfin-theater-windows)** - A desktop media interface for Windows.
+**[Jellyfin Theater (Electron)](https://github.com/jellyfin/jellyfin-theater-electron)**
+
+A desktop media interface for Windows, macOS, and Linux.
+Status: No Maintainers
```

Comment:
That is not completely true, there are a few guys doing things. Not anything nearing a release, but still...